<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CFDModelPostprocessing</class>
 <widget class="qMRMLWidget" name="CFDModelPostprocessing">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>563</width>
    <height>574</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="ctkCollapsibleButton" name="loadFileCollapsibleButton">
     <property name="text">
      <string>Load File</string>
     </property>
     <layout class="QFormLayout" name="formLayout_2">
      <item row="1" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Select file:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="ctkPathLineEdit" name="filePathLineEdit">
        <property name="filters">
         <set>ctkPathLineEdit::Executable|ctkPathLineEdit::Files|ctkPathLineEdit::NoDot|ctkPathLineEdit::NoDotDot|ctkPathLineEdit::Readable</set>
        </property>
        <property name="nameFilters">
         <stringlist>
          <string>*_tec.dat</string>
          <string>*.tec</string>
          <string>*.vtp</string>
          <string>*.vtk</string>
          <string>*.stl</string>
          <string>*.ply</string>
          <string>*obj</string>
          <string>*.vtu</string>
          <string></string>
         </stringlist>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QCheckBox" name="scaleFactorCheckBox">
        <property name="text">
         <string>Scale geometry:</string>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="scaleFactorLineEdit">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>1000</string>
        </property>
        <property name="maxLength">
         <number>32771</number>
        </property>
        <property name="cursorPosition">
         <number>4</number>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QCheckBox" name="lpsToRasCheckBox">
        <property name="text">
         <string>LPS to RAS</string>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="4" column="0" colspan="2">
       <widget class="QPushButton" name="loadButton">
        <property name="text">
         <string>Load</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="qMRMLCollapsibleButton" name="MRMLCollapsibleButton">
     <property name="text">
      <string>Dsiplay model surface</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="qMRMLScalarsDisplayWidget" name="surfaceVariablesDisplayWidget">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="ctkCollapsibleButton" name="computeCenterlinesCollapsibleButton">
     <property name="text">
      <string>Compute centerline</string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Endpoints:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="qMRMLNodeComboBox" name="endPointsMarkupsSelector">
          <property name="toolTip">
           <string>Branch endpoints. &quot;Unselected&quot; control points are used as sources, &quot;selected&quot; control points are used as targets.</string>
          </property>
          <property name="nodeTypes">
           <stringlist notr="true">
            <string>vtkMRMLMarkupsFiducialNode</string>
           </stringlist>
          </property>
          <property name="hideChildNodeTypes">
           <stringlist notr="true"/>
          </property>
          <property name="interactionNodeSingletonTag">
           <string notr="true"/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="qSlicerMarkupsPlaceWidget" name="endPointsMarkupsPlaceWidget"/>
        </item>
        <item>
         <widget class="ctkPushButton" name="autoDetectEndPointsPushButton">
          <property name="font">
           <font>
            <pointsize>8</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Auto-detect</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="2" column="0">
       <widget class="QPushButton" name="computeCenterlineButton">
        <property name="text">
         <string>Compute centerline</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="QGroupBox" name="selectInletGroupBox">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="title">
         <string>Select inlet:</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ctkCollapsibleButton</class>
   <extends>QWidget</extends>
   <header>ctkCollapsibleButton.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ctkPathLineEdit</class>
   <extends>QWidget</extends>
   <header>ctkPathLineEdit.h</header>
  </customwidget>
  <customwidget>
   <class>ctkPushButton</class>
   <extends>QPushButton</extends>
   <header>ctkPushButton.h</header>
  </customwidget>
  <customwidget>
   <class>qMRMLCollapsibleButton</class>
   <extends>ctkCollapsibleButton</extends>
   <header>qMRMLCollapsibleButton.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>qMRMLNodeComboBox</class>
   <extends>QWidget</extends>
   <header>qMRMLNodeComboBox.h</header>
  </customwidget>
  <customwidget>
   <class>qMRMLScalarsDisplayWidget</class>
   <extends>qMRMLWidget</extends>
   <header>qMRMLScalarsDisplayWidget.h</header>
  </customwidget>
  <customwidget>
   <class>qMRMLWidget</class>
   <extends>QWidget</extends>
   <header>qMRMLWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>qSlicerWidget</class>
   <extends>QWidget</extends>
   <header>qSlicerWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>qSlicerMarkupsPlaceWidget</class>
   <extends>qSlicerWidget</extends>
   <header>qSlicerMarkupsPlaceWidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>CFDModelPostprocessing</sender>
   <signal>mrmlSceneChanged(vtkMRMLScene*)</signal>
   <receiver>endPointsMarkupsPlaceWidget</receiver>
   <slot>setMRMLScene(vtkMRMLScene*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>156</x>
     <y>273</y>
    </hint>
    <hint type="destinationlabel">
     <x>373</x>
     <y>429</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>CFDModelPostprocessing</sender>
   <signal>mrmlSceneChanged(vtkMRMLScene*)</signal>
   <receiver>endPointsMarkupsSelector</receiver>
   <slot>setMRMLScene(vtkMRMLScene*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>156</x>
     <y>273</y>
    </hint>
    <hint type="destinationlabel">
     <x>245</x>
     <y>429</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>endPointsMarkupsSelector</sender>
   <signal>currentNodeChanged(vtkMRMLNode*)</signal>
   <receiver>endPointsMarkupsPlaceWidget</receiver>
   <slot>setCurrentNode(vtkMRMLNode*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>245</x>
     <y>429</y>
    </hint>
    <hint type="destinationlabel">
     <x>373</x>
     <y>429</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>CFDModelPostprocessing</sender>
   <signal>mrmlSceneChanged(vtkMRMLScene*)</signal>
   <receiver>surfaceVariablesDisplayWidget</receiver>
   <slot>setMRMLScene(vtkMRMLScene*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>281</x>
     <y>286</y>
    </hint>
    <hint type="destinationlabel">
     <x>281</x>
     <y>313</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
